operations-stdlib/src/commonMain/kotlin/Length.kt

```diff
@@ -0,0 +1,12 @@
+import utils.asList
+import operation.StandardLogicOperation
+
+object Length : StandardLogicOperation {
```

Comment:
Maybe in operations-stdlib we should add our own KOtlin-like StringUnwrapStrategy to not repeat code that unwraps expression to string? 
I mean something like this:
https://github.com/allegro/json-logic-kmp/blob/operators-lowercase-uppercase-capitalize/operations-stdlib/src/commonMain/kotlin/string/StringUnwrapStrategy.kt
And then code in operator simplifies to this:
https://github.com/allegro/json-logic-kmp/blob/operators-lowercase-uppercase-capitalize/operations-stdlib/src/commonMain/kotlin/string/Lowercase.kt

WDYT? @mkrogulskiallegro2 @KTGypsy

Reply:
copied your code to same place. Should be compatible now